src/class/Context/contextHandlers.js

```diff
@@ -1,91 +1,156 @@
+// @ts-check
 import ConnectError from '@/class/ConnectError';
 import { PLUGIN_METHODS, MESSENGER_METHODS, PLUGIN_NAMES } from '@/constants';
 
 const { ERRORS } = ConnectError;
 
-const initiate = context => (payload, req) => {
-  const { isIdentityMode } = context.options;
-
-  req.answer({
-    isIdentityMode: isIdentityMode || false,
-  });
-};
-
-const changeSettings = context => async (payload, req) => {
-  try {
-    await context.executeMethod(
-      PLUGIN_METHODS.CONTEXT_SET_PROVIDER_SETTINGS,
-      payload,
-    );
-
+/** 
+ * @param {Context} context 
+ * 
+ * @returns {RequestEventHandler}
+ */
+const initiate = context => {
+  return (payload, req) => {
```

Comment:
please, pass all changed code with eslint, i just fetched your branch and have a lot of eslint errors formating =(

Reply:
My code always formatted very well. Very strange, check it one more time, thanks